keepassxc-browser/content/ui.js

```diff
@@ -85,7 +85,12 @@ kpxcUI.setIconPosition = function(icon, field) {
     const size = Number(icon.getAttribute('size'));
 
     icon.style.top = Pixels((rect.top + document.scrollingElement.scrollTop) + offset + 1);
-    icon.style.left = Pixels((rect.left + document.scrollingElement.scrollLeft) + field.offsetWidth - size - offset);
+    if (document.dir == 'rtl') {
```

Comment:
FYI: proper strict JavaScript comparison is `=== `.

Reply:
You're right! I updated it. Thanks for the prompt feedback.